### ISSUE TITLE
Command line can't render any project

### ISSUE THREAD

**Issue by karmux** [LMMS/lmms#3434]
When project has VST plugins:
```
$ ./lmms -r ~/Documents/LMMS/projects/ProjectWithVST.mmpz -o ~/ProjectWithVST.ogg -f ogg
VST sync support disabled in your configuration
Loading project...
QWidget: Cannot create a QWidget without QApplication
Aborted (core dumped)
```

When no VST plugins:
```
$ ./lmms -r ~/Documents/LMMS/projects/ProjectWithoutVST.mmpz -o ~/ProjectWithoutVST.ogg -f ogg
VST sync support disabled in your configuration
Loading project...
Segmentation fault (core dumped)
```

When project has just Kicker, it freezes:
```
$ ./lmms -r ~/Documents/LMMS/projects/KickerProject.mmpz -o ~/KickerProject.ogg -f ogg
VST sync support disabled in your configuration
Loading project...
Done
|-----------------------------------------------   |     92%   /
```

I can render them using GUI.

LMMS master on Ubuntu 16.10.

**Comment by zonkmachine** [comment]
> When project has VST plugins:

I get this too. Both in master, stable-1.1.3 and stable-1.0.3

> When no VST plugins:
> ...
> When project has just Kicker, it freezes:

It works fine here. Did you apply any changes to the software?
You can test this with `git --diff`, it should give no output.
What is your output from `./lmms --version` ?
What audio interface are you using?

**Comment by karmux** [comment]
I have no changes to software.
```
./lmms --version
LMMS 1.2.0-rc2.95
(Linux x86_64, Qt 5.6.1, GCC 6.2.0 20161005)
```
I'm using SDL audio.

**Comment by zonkmachine** [comment]
> I have no changes to software.
> 
> ```
> ./lmms --version
> LMMS 1.2.0-rc2.95
> ```

I'm at the latest master and if I'm interpreting the output correctly I'm 6 commits after you.
`LMMS 1.2.0-rc2.89`

With `git log` , do you get the same output, commits and version SHA, as this page?
https://github.com/LMMS/lmms/commits/master

**Comment by karmux** [comment]
Yes `git log` is same.

**Comment by PaulBatchelor** [comment]
@karmux It may be an unrelated issue, but is there any chance you could try my PR #3404?

**Comment by karmux** [comment]
@PaulBatchelor your PR fixes rendering projects without VSTs :+1: 
Even project having GigaSampler rendered nicely.
I think this issue can be closed once your PR gets merged. VST command line rendering feels like a separate bigger issue.

**Comment by Lukas-W** [comment]
Fixed via @PaulBatchelor's #3404.

**Comment by Lukas-W** [comment]
Opened a new issue for VST plugin rendering: #3449
